IntegrationExample/Processing/View/ProcessingView.swift

```diff
@@ -6,17 +6,44 @@
 //
 
 import SwiftUI
+import Combine
 import MimiCoreKit
 
 struct ProcessingView: View {
     
+    @EnvironmentObject var headphoneConnectivity: PartnerHeadphoneConnectivityController
     @ObservedObject var viewModel: ProcessingViewModel
+
+    @State var isHeadphoneConnected: Bool = true
```

Comment:
Remove the extra line and make the properties private.
```suggestion
    @EnvironmentObject private var headphoneConnectivity: PartnerHeadphoneConnectivityController
    @ObservedObject private var viewModel: ProcessingViewModel
    @State private var isHeadphoneConnected: Bool = true
```